docs/wcs/index.rst

```diff
@@ -229,30 +229,30 @@ area can be extracted with the utility function
 Matplotlib plots with correct WCS projection
 ============================================
 
-The `WCSAxes <http://wcsaxes.readthedocs.io>`_ affiliated package adds the
-ability to use the :class:`~astropy.wcs.WCS` to define projections in
-Matplotlib. More information on installing and using WCSAxes can be found `here
-<http://wcsaxes.readthedocs.io>`__.
+The :ref:`WCSAxes <wcsaxes>` framework, now included in Astropy, allows the
```

Comment:
Remove the "now included in Astropy" (or replace it with: "previously a stand-alone pacakge").